packages/slice-machine/src/hooks/useMarketingContent.ts

```diff
@@ -17,6 +17,14 @@ type MarketingContent = {
       };
     };
   };
+  masterSliceLibrary: MasterSliceLibraryMarketingContent;
```

Comment:
I love the idea! Super nice, and aligned with our objectives. No direct content written in the component 🙌

I just think we could keep the same structure with:

```suggestion
  masterSliceLibrary?: {
  exampleLinkUrl: string;
  codeLinkUrl: string;
  previewVideoUrl: string;
};
```

Like others, no need of showInNavigation boolean, you just check that the object is defined.

Reply:
Changed